src/components/onboardingGuide/OnboardingGuide.tsx

```diff
@@ -54,6 +53,16 @@ function OnboardingGuide({
         handlePostHogEventUpdate(e, POSTHOG_EVENT_ONBOARDING.PREVIEW)
     }
 
+    const handleSkipOnboarding = () => {
+        if (loginCount === 0) {
```

Comment:
why this login count check is here? it should anyway be visible on first load

Reply:
What if we are going in the `Getting started` page via `Help`section?